### PR TITLE
Replace SE by PNN in the recovery script

### PR DESCRIPTION
**Pull request by amaltaro**
@prozober please review and test. First commit has important changes, 2nd one is only aesthetic and convention fixes.

### CODE REVIEW

File: recoverMissingLumis.py [hunk 1]
```diff
@@ -217,86 +221,81 @@ def getFiles(datasetName, runBlacklist, runWhitelist, blockBlacklist,
...
-                node = replica["node"]
-                cmsSites = siteDB.PNNtoPSN(node)
-                if type(cmsSites) != list:
-                    cmsSites = [cmsSites]
-                for cmsName in cmsSites:
-                    se = siteDB.cmsNametoSE(cmsName)
-                    blockLocations.update(se)
-                    logging.debug("cmsName %s mapped to se %s", cmsName, se)
-                logging.debug("PhEDEx node %s, cmsSites %s, blockLocations %s", node, cmsSites, blockLocations)
+                PNN = replica["node"]
```
Comment (vlimant) [inline recoverMissingLumis.py]: that and the line below is the only actual change, correct ?

Comment (amaltaro) [inline recoverMissingLumis.py]: See the first commit only. There were 3 fixes, 2 regarding SE stuff and one regarding a new change in the ACDC Server code.

**Comment by prozober** [comment]
Hi Alan, we have a problem:

```
Traceback (most recent call last):
  File "recoverMissingLumis.py", line 698, in <module>
    sys.exit(main())
  File "recoverMissingLumis.py", line 694, in main
    val.group, val.dbsUrl, val.fake, datasetInformation)
  File "recoverMissingLumis.py", line 561, in defineRequests
    fileset.setCollection(collection)
  File "/data/srv/wmagent/v1.1.0.patch2/sw/slc6_amd64_gcc493/cms/wmagent/1.1.0.patch2/lib/python2.7/site-packages/WMCore/ACDC/Fileset.py", line 34, in setCollection
    self.owner = collection.owner
AttributeError: 'CouchCollection' object has no attribute 'owner'
```

**Comment by ticoann** [comment]
@prozober, Paola, could you update the source code to 1.1.1.pre9 wmagent?

**Comment by amaltaro** [comment]
Paola is using vocms049 (unified node) to run this script and it of course doesn't have the patches we create :-) Anyways I patched that agent and the script was apparently running Ok. That node has 1.1.0.patch2, so it should be fine.

**Pull request merged**